Library/Formula/git-plus.rb

```diff
@@ -0,0 +1,25 @@
+class GitPlus < Formula
+  homepage "https://github.com/tkrajina/git-plus"
+  head "https://github.com/tkrajina/git-plus.git"
+  url "https://github.com/tkrajina/git-plus/archive/v0.1.0.tar.gz"
+  sha256 "d7076ed3fd8e6e4bc2b93b9880adf58bea59b0f6fe95e05053f00247ae8cb609"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    site_packages = libexec/"lib/python2.7/site-packages"
+    libexec_bin = libexec/"bin"
+
+    ENV.prepend_create_path("PYTHONPATH", site_packages)
+
+    site_packages.install("gitutils.py")
+    libexec_bin.install(Dir["git-*"])
+
+    bin.install(Dir[libexec_bin/"*"])
+    bin.env_script_all_files(libexec_bin, :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    shell_output("git multi")
```

Comment:
Testing:

```ruby
  test do
    system "git", "multi"
  end
```

```
~> brew test --verbose /Users/Dominyk/Downloads/git-plus.rb
Testing git-plus
==> git multi
--------------------------------------------------------------------------------
Executing git status -s
--------------------------------------------------------------------------------
```
===
```ruby
  test do
    mkdir "testme" do
      system "git", "init"
      touch "README"
      system "git", "add", "README"
      system "git", "commit", "-m", "testing"
      rm "README"
    end

    assert_match /D README/, shell_output("#{bin}/git-multi")
  end
```

```
Testing git-plus
==> git init
Initialized empty Git repository in /private/tmp/git-plus20150323-7224-1xj0fp/testme/.git/
==> git add README
==> git commit -m testing
[master (root-commit) a69f4ac] testing
 Committer: Dominyk Tiller <Dominyk@Dominyks-MacBook-Pro.local>
Your name and email address were configured automatically based
on your username and hostname. Please check that they are accurate.
You can suppress this message by setting them explicitly. Run the
following command and follow the instructions in your editor to edit
your configuration file:

    git config --global --edit

After doing this, you may fix the identity used for this commit with:

    git commit --amend --reset-author

 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 README
==> /usr/local/Cellar/git-plus/0.1.0/bin/git-multi
```

Whichever test chosen should work. Clean exit codes everywhere.